docs-chef-io/content/inspec/resources/docker_container.md

```diff
@@ -40,18 +40,18 @@ A `docker_container` resource block declares the configuration data to be tested
 
 ## Resource Parameter Examples
 
-### name
+### Name
 
-The container name can also be passed with the `name` resource parameter:
+The container name can be provided with the `name` resource parameter.
 
     describe docker_container(name: 'an-echo-server') do
       it { should exist }
       it { should be_running }
     end
 
-### id
+### Container ID
```

Comment:
This should be `id` and not `Container ID` since Ruby identifiers are case sensitive and `id` is the assigned identifier instead of `Container ID`